README.md

```diff
@@ -0,0 +1,205 @@
+# edX ESLint config
+
+ESLint config for edX JavaScript code.
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [License](#license)
+3. [Dependencies](#dependencies)
+4. [Usage](#usage)
+5. [The Config](#the-config)
+
+## Overview
+
+In order to standardize and enforce edX's JavaScript coding style across multiple codebases, edX has adopted ESLint and this shared ESLint config. This document describes the edX JavaScript style and its inspirations.
+
+## License
+
+The code in this repository is licensed the Apache 2.0 license unless otherwise
```

Comment:
Yeah, thoughts on that? I just added that because the Toolkit is Apache. [Technically there's only one line of code in this repo](https://github.com/edx/eslint-config-edx/blob/bjacobel/feedback/index.js#L1), is Apache the right choice for that situation? The Airbnb styleguide / linter config this extends is MIT-licensed.